requirements.txt

```diff
@@ -1,28 +1,29 @@
 # app requirements
 CairoSVG==2.2.1
+
 celery[sqs]==4.3.0
-Flask==1.0.3
-Flask-WeasyPrint==0.5
-Flask-HTTPAuth==3.2.4
+Flask==1.1.1
+Flask-WeasyPrint==0.6
+Flask-HTTPAuth==3.3.0
 html5lib==1.0.1
-wand==0.4.4
-jsonschema==3.0.1
+wand==0.5.8
+jsonschema==3.2.0
 pypdf2==1.26.0
-Pillow==5.2.0
-reportlab==3.5.6
-pdf2image==0.1.14
-PyMuPDF==1.16.1
-Werkzeug==0.14.1  # pyup: < 0.15.0
+Pillow==6.2.1
+reportlab==3.5.32
+pdf2image==1.10.0
+PyMuPDF==1.16.9
+Werkzeug==0.16.0
 pdfrw==0.4
 defusedxml==0.6.0
+WeasyPrint==50
 
-
-# weasyprint 0.41 has errors where png previews of PDFs randomly do not render all images
-# weasyprint 0.42 has errors where it gets stuck in an infinite loop and we had to kill jenkins after 7 hrs
-WeasyPrint==0.40  # pyup: != 0.41, != 0.42
-
-git+https://github.com/alphagov/notifications-utils.git@35.0.0#egg=notifications-utils==35.0.0
+git+https://github.com/alphagov/notifications-utils.git@36.3.1#egg=notifications-utils==36.3.1
 
 # PaaS requirements
-gunicorn==19.9.0
+gunicorn==20.0.4
 awscli-cwlogs>=1.4,<1.5
+
+# CairoSVG-2.4.2
+# Flask-1.1.1
+# Werkzeug-0.16.0
```

Comment:
Why are these commented out?

Flask and Werkzeug seem to be duplicates of definitions above. The commented out version of CairoSVG is newer than the one referenced above.

Reply:
it's been a while since i looked at this but iirc these versions were crashing. i might take a look at this PR again later today/tomorrow. either way these commented lines shouldn't really be here.